includes/ucf-social-config.php

```diff
@@ -0,0 +1,339 @@
+<?php
+/**
+ * Handles plugin configuration
+ */
+
+if ( !class_exists( 'UCF_Social_Config' ) ) {
+
+	class UCF_Social_Config {
+		public static
+			$option_prefix = 'ucf_social_',
+			$option_defaults = array(
+				'include_css' => true,
```

Comment:
Default values for facebook_url, twitter_url, etc need to be set here for most of the option-or-default methods to work correctly--just setting the defaults to an empty string will work.